### ISSUE TITLE
Unobserved Task Exception

### ISSUE THREAD

**Issue by andrewmd5** [deniszykov/WebSocketListener#17]
```
WebSocketException: Write operation on WebSocket stream is failed: Unable to access a deleted object.
Name of the object: 'SslStream'.
  Module "vtortola.WebSockets.Rfc6455.WebSocketConnectionRfc6455+<SendFrameAsync>d__43", line 900, col 0, in MoveNext
    Void MoveNext()
ObjectDisposedException: Can not access a deleted object.
Name of the object: 'SslStream'.
  Module "vtortola.WebSockets.Rfc6455.WebSocketConnectionRfc6455+<SendFrameAsync>d__43", line 483, col 0, in MoveNext
    Void MoveNext()
  Module "System.Runtime.CompilerServices.TaskAwaiter", line 40, col 0, in HandleNonSuccessAndDebuggerNotification
    Void HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
  Module "System.Runtime.ExceptionServices.ExceptionDispatchInfo", line 12, col 0, in Throw
    Void Throw()
  Module "System.Threading.Tasks.TaskFactory`1+FromAsyncTrimPromise`1", line 2, col 0, in Complete
    Void Complete(TInstance, System.Func`3[TInstance,System.IAsyncResult,TResult], System.IAsyncResult, Boolean)
  Module "System.IO.Stream+<>c", line 7, col 0, in <BeginEndWriteAsync>b__53_1
    System.Threading.Tasks.VoidTaskResult <BeginEndWriteAsync>b__53_1(System.IO.Stream, System.IAsyncResult)
  Module "System.Net.Security.SslStream", line 0, col 0, in EndWrite
    Void EndWrite(System.IAsyncResult)
  Module "System.Net.Security.SslState", line 8, col 0, in get_SecureStream
    System.Net.Security._SslStream get_SecureStream()
  Module "System.Net.Security.SslState", line 41, col 0, in CheckThrow
    Void CheckThrow(Boolean, Boolean)
```

**Comment by andrewmd5** [comment]
Saw you fixed this, anyway we can get an alpha build on nuget for testing? Thanks for the quick response.

**Comment by deniszykov** [comment]
Hi Andrew,
I think I fixed it in 4.1.6-alpha. I all be ok, I will release it as 4.2.0.
